app/views/admin/teachers/show.html.erb

```diff
@@ -13,7 +13,3 @@
 <%= render(Teachers::PastInductionPeriodsComponent.new(teacher: @teacher)) %>
 
 <%= govuk_warning_text(text: "Some of this teacher's records could not be migrated") if @teacher.has_migration_failures? %>
-
-<p class="govuk-body">
-  <%= govuk_link_to "Migration detail", migration_teacher_path(@teacher, page: @page) %>
-</p>
```

Comment:
@tonyheadford - this can be re-added but we'll need it behind an environment variable flag or something so worth doing that as a separate PR.